src/procedure/nodes/sum1d.cpp

```diff
@@ -93,7 +94,10 @@ bool Sum1DProcedureNode::finalise(const ProcedureContext &procedureContext)
                 sum_[i] = procedureContext.dataList().realise<SampledDouble>(
                     fmt::format("Sum1D//{}//{}", name(), rangeNames[i]), procedureContext.dataPrefix(),
                     GenericItem::InRestartFileFlag);
-            sum_[i]->get() += Integrator::sum(sourceData_->processedData(), range_[i]);
+            if (instantaneous_)
+                sum_[i]->get() = Integrator::sum(sourceData_->processedData(), range_[i]);
+            else
+                sum_[i]->get() += Integrator::sum(sourceData_->processedData(), range_[i]);
```

Comment:
```suggestion
            sum_[i]->get() = Integrator::sum(sourceData_->processedData(), range_[i]) + (instantaneous_ ? 0 : sum_[i]->get());
```

A possible suggestion for reducing some duplication.

Reply:
This unfortunately loses information as the `=` operator resets the `SampledDouble` to a single data point (i.e. count of 1) while the `+=` accumulates the average and standard deviation while absorbing the value.